### PR TITLE
luci-app-xinetd: add new package

### PR DESCRIPTION
**Pull request by he-ma**
This is a luci application to configure xinetd services.
Only the (probably) most used parameters are implemented for now.

This package depends on PR in another repo (https://github.com/openwrt/packages/pull/12389)

### CODE REVIEW

File: applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+		o.default = 'tcp';
+		o.value('tcp', _('TCP'));
+		o.value('udp', _('UDP'));
+		o.rmempty = false;
+		o.modalonly = true;
+
+		o = s.taboption('basic', form.ListValue, 'user', _('User (UID)'), _('User ID for the server process for this service'));
+		o.rmempty = false;
+		o.modalonly = true;
+		o.load = function(section_id) {
```
Comment (jow-) [inline applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js:117]: I suggest to use `widgets.UserSelect` here. This way you can get rid of the custom `load()` and `getUsers()` logic.

Comment (he-ma) [reply applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js:117]: Thank you for your hint. I will modify it.
But, do you have a tipp how to clear the (existing) keylist if not clearing it in a custom load()?
The list get's longer every time as the user list is appended to it.
Thanks in advance.

Comment (jow-) [reply applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js:117]: That is probably a bug we should fix separately in `widgets.js`

Comment (he-ma) [reply applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js:117]: OK.
What do you mean, should I wait for the fix or maybe doing a workaround wit a load() function which resets the list by doing a `this.keylist = 0` only (as I already did)

**Comment by he-ma** [comment]
@jow- Sorry for the delay in responding, but I had been very busy.
Now I made the change you requested (and some other improvements which probably wouldn't allow this app to get upstream).

**Comment by jow-** [comment]
Merged, thank you!

**Pull request merged**